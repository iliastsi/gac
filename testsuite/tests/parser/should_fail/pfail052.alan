MulDimArray() : proc
  i : int [4][5];
{}
