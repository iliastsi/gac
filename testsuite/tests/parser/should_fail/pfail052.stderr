pfail052.alan:2:17: Error: Parse error on input `;'
	Alan doesn't support arrays bigger than one dimension
	Use -XMultiDimArrays if you want to enable this

